doc.go

```diff
@@ -455,6 +455,41 @@ of the returned value:
 	    }
 	}
 
+## Arrow batches
+
+There is a possibility to retrieve data in a columnar format similar to native server response.
+You have to use a proper `WithArrowBatches` context, example:
+
+	    var rows driver.Rows
+		err = conn.Raw(func(x interface{}) error {
+			rows, err = x.(driver.QueryerContext).QueryContext(ctx, query, nil)
+			return err
+		})
+
+		...
+
+		batches, err := rows.(sf.SnowflakeRows).GetArrowBatches()
+
+		... // use arrow records
+
+Limitations:
+
+1. When a query regards something that is not real data, it is returned usually in JSON format. It can't be used with arrow batches context. Examples: `SHOW PARAMETERS`.
+2. Snowflake handles timestamps in a range which is higher than available space in Arrow timestamp type. Because of that special treatment should be used (see below).
+
+### Handling timestamps in Arrow batches
+
+Snowflake returns timestamps natively (from backend to driver) in multiple formats.
+Arrow timestamp type is only 8 bytes long and is not enough to cover the whole date/time Range Snowflake handles.
```

Comment:
The Arrow timestamp is an 8-byte data type, which is insufficient to handle the larger date and time ranges used by Snowflake.